tests/rptest/services/admin_ops_fuzzer.py

```diff
@@ -222,14 +224,44 @@ def __init__(self, prefix):
         self.total = None
         self.current = None
 
+    def _current_partition_count(self, ctx):
+
+        per_node_count = set()
+        for n in ctx.redpanda.started_nodes():
+            node_version = int_tuple(
+                VERSION_RE.findall(ctx.redpanda.get_version(n))[0])
+            # do not query nodes with redpanda version prior to v23.1.x
+            if node_version[0] < 23:
+                return None
+            try:
+                partitions = ctx.admin().get_partitions(node=n,
+                                                        topic=self.topic)
+                per_node_count.add(len(partitions))
+            except HTTPError as err:
+                if err.response.status_code == 404:
+                    return None
+                else:
+                    raise
+
+        if len(per_node_count) != 1:
```

Comment:
nit: Add more logging, which node has different partition amount or partition amount for every node